### ISSUE TITLE
Argument list too long: recursive header expansion failed - IOS

### ISSUE THREAD

**Issue by Poyser1911** [apptentive/apptentive-react-native#59]
Setup:
1. React-native: 0.57.8
2. Followed `installation` instructions for `ios only` using `cocoaPods` and `manual install` of the `.xcodeproj` file `without` the code involving `Usage` (adding api key etc.)
My Pod File:
```
platform :ios, '10.0' // only change from the docs with uses v9

target ' [App Name]' do
  pod 'TrustKit'

  pod 'apptentive-ios'
```

When ever i try to clean / Run project in XCode i get this same error in the issue navigator tab:

`RNApptentiveModule 1 issue`

- `Dependency Analysis Error`

- `Argument list too long: recursive header expansion failed at [App Path]/node_modules/apptentive-react-native/ios/../../../ios/Pods/React/node_modules/resolve/test/resolver/dot_slash_main`

**Comment by frankus** [comment]
We're taking a look at this today. 

It sounds like there may be an issue with the Header Search Paths setting where it isn't specific enough and is recursing through a bunch of unrelated node modules.

**Comment by Poyser1911** [comment]
@frankus 
Here is my project header search paths:
![Screen Shot 2019-03-26 at 1 32 46 PM](https://user-images.githubusercontent.com/11511131/55024034-09ed9a80-4fcc-11e9-824f-4082d2020f0d.png)

**Comment by frankus** [comment]
Would you be willing to try an edit to the RNApptentiveModule project? In the Header Search Paths, replace:
```
$(SRCROOT)/../../../ios/Pods/**
```
with:
```
$(SRCROOT)/../../../ios/Pods/apptentive-ios/Apptentive/**
```
Let me know if that works and if so we can patch our module so that you won't have to make this edit in the future.

**Comment by Poyser1911** [comment]
In Libraries -> RNApptentiveModule.xcodeproj -> Build Settings -> Search Paths -> Header Search Paths -> [Its empty] :/
Tried adding those, same error

**Comment by Poyser1911** [comment]
Doing some refactor to get to the latest RN, will let you know if it works

**Comment by Poyser1911** [comment]
Seems to be working on RN 0.59.2 fine.

**Comment by shercoder** [comment]
Hi - Is there a fix for someone who hasn't upgraded to RN 0.59.2? I'm on 0.57.3.
